jest-environment-node missing

Trying to add to https://github.com/artsy/elderfield/pull/42

```

> elderfield@0.0.1 danger /home/travis/build/artsy/elderfield
> danger
module.js:327
    throw err;
    ^
Error: Cannot find module 'jest-environment-node'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/travis/build/artsy/elderfield/node_modules/danger/distribution/runner/DangerfileRunner.js:131:28)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
```

--- a/changelog.md
+++ b/changelog.md
@@ -3,6 +3,7 @@
 //  Add your own contribution below
 
 * Add summary comment for danger message - kwonoj
+* Add `jest-environment-node` to the Package.json - orta
 
 ### 0.9.0
 

--- a/package.json
+++ b/package.json
@@ -98,6 +98,7 @@
     "babel-polyfill": "^6.20.0",
     "commander": "^2.9.0",
     "debug": "^2.6.0",
+    "jest-environment-node": "^18.1.0",
     "jest-runtime": "^18.0.0",
     "lodash.find": "^4.6.0",
     "lodash.includes": "^4.3.0",

